qutebrowser/commands/runners.py

```diff
@@ -195,6 +196,8 @@ def parse(self, text, *, fallback=False, keep=False):
             return ParseResult(cmd=None, args=None, cmdline=cmdline)
 
         args = self._split_args(cmd, argstr, keep)
+        args = [x.strip().lstrip(':').strip() if x is not ':' and x is not ' '
+                else x for x in args]
```

Comment:
I don't think it makes sense to apply this on the arguments.

Reply:
Hmm...why not?